node-notes-cli/app.js

```diff
@@ -0,0 +1,45 @@
+const fileSys = require('fs');
+const command = process.argv[2];
+const input = process.argv[3];
+const input2 = process.argv[4];
+const obj = require('./data.json');
+
+if (command === 'read') {
+  for (var i = 1; i < obj.nextId; i++) {
+    console.log([i].toString() + ':', obj.notes[i]);
+  }
```

Comment:
Try doing this with a for ... in loop to avoid printing undefined for missing entries.

Reply:
```js
for (const id in obj.notes) {

}
```